langs/arturo/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.20 AS builder
+
+RUN apk add --no-cache build-base curl git nim pcre
+
+RUN git clone https://github.com/arturo-lang/arturo.git
```

Comment:
This still needs to be pinned, if tarballs don't work (for whatever reason) then we need to check out a versioned branch/tag instead, `git clone -b` is your friend.

Reply:
Already tried, same misery as tarball.

Reply:
So older versions can't be built? Well we'll need to pick a recent sha if that's the case, we can't have it unpinned or we'd get a different version every time it was built.

Reply:
> So older versions can't be built?

Apparently, no. I'll try 0.9.82/80, and see what that brings. Regardless, I want 0.9.83 or it ain't happening, at least not by me.

Reply:
It's all happened.